pkg/webhook/multiclusterservice/validating.go

```diff
@@ -45,7 +45,7 @@ var _ admission.Handler = &ValidatingAdmission{}
 
 // Handle implements admission.Handler interface.
 // It yields a response to an AdmissionRequest.
-func (v *ValidatingAdmission) Handle(ctx context.Context, req admission.Request) admission.Response {
+func (v *ValidatingAdmission) Handle(_ context.Context, req admission.Request) admission.Response {
```

Comment:
I mean should I also remove the other parameters like this one? @RainbowMango

Reply:
I'm afraid we can't remove the unused parameters for those functions that implement some interfaces.
Take this function, for example, it implements the interface:
```golang
// Handler can handle an AdmissionRequest.
type Handler interface {
	// Handle yields a response to an AdmissionRequest.
	//
	// The supplied context is extracted from the received http.Request, allowing wrapping
	// http.Handlers to inject values into and control cancelation of downstream request processing.
	Handle(context.Context, Request) Response
}
```
So, you can't remove them, all what we can do is rename it to `_`.